api/models/CollectionsPost.js

```diff
@@ -0,0 +1,37 @@
+module.exports = bookshelf.Model.extend({
+  tableName: 'collections_posts',
+  requireFetch: false,
+  hasTimestamps: true,
+
+  collection: function () {
+    return this.belongsTo(Collection, 'collection_id')
```

Comment:
Just a note: I think that ES6 module `import` mostly just works currently on `hylo-node`. Not for this PR, but going forward it might be nice if we start importing our dependencies in the way we're accustomed to in Web, etc. vs relying on the global scaffolding...